jax/test_util.py

```diff
@@ -52,6 +52,14 @@
   int(os.getenv('JAX_NUM_GENERATED_CASES', 10)),
   help='Number of generated cases to test')
 
+flags.DEFINE_integer(
+  'max_retries',
+  int(os.getenv('JAX_MAX_RETRIES', 100)),
```

Comment:
Perhaps the name could be less generic, e.g., 'JAX_MAX_CASES_SAMPLING_RETRIES'?

Reply:
Good point. Updated with your suggestion.